dev/storage.h

```diff
@@ -124,16 +124,39 @@ namespace sqlite_orm {
                 }
                 perform_void_exec(db, ss.str());
             }
-
+#if 0  // jdh
```

Comment:
are you sure that this code is needed?

Reply:
eliminated and no copy_table_from does not exist anymore - check the latest commit please!